config/gtasa/constants.xml

```diff
@@ -58,30 +58,34 @@
       <Constant Name="LEFTSHOCK"/>
       <Constant Name="RIGHTSHOCK"/>
     </Enum>
-    <!-- Seems to be the same as VC -->
     <Enum Name="FACING">
       <Constant Name="FACING_FORWARD"/>
       <Constant Name="FACING_LEFT"/>
       <Constant Name="FACING_BACK"/>
       <Constant Name="FACING_RIGHT"/>
     </Enum>
-    <!-- Values according to http://gtag.gtagaming.com/opcode-database/opcode/04E3/ -->
     <Enum Name="PLAYER_MOOD">
-      <Constant Name="PLAYER_MOOD_CALM" Value="5"/>
-      <Constant Name="PLAYER_MOOD_PISSED_OFF" Value="7"/>
-      <Constant Name="PLAYER_MOOD_ANGRY" Value="1"/>
-      <Constant Name="PLAYER_MOOD_WISECRACKING" Value="9"/>
```

Comment:
I'm going to create a `legacy-constants.xml`  or something and keep these for `--guesser` mode such that existing scripts don't break.

Reply:
Yes, probably. I originally wanted to add only the constants that were not added here, but then I found out that many of the already existing constants had different names, so I decided to make them exactly the same as in sc.exe